src/databricks/sql/auth/oauth.py

```diff
@@ -96,24 +100,25 @@ def __get_authorization_code(self, client, auth_url, scope, state, challenge):
                         code_challenge=challenge,
                         code_challenge_method="S256",
                     )
-                    logger.info(f"Opening {auth_req_uri}")
+                    logger.info("Opening %s", auth_req_uri)
 
                     webbrowser.open_new(auth_req_uri)
                     logger.info(
-                        f"Listening for OAuth authorization callback at {redirect_url}"
+                        "Listening for OAuth authorization callback at %s", redirect_url
                     )
                     httpd.handle_request()
                 self.redirect_port = port
                 break
             except OSError as e:
                 if e.errno == 48:
-                    logger.info(f"Port {port} is in use")
+                    logger.info("Port %d is in use", port)
                     last_error = e
             except Exception as e:
-                logger.error("unexpected error", e)
```

Comment:
This is probably the issue that brought me here in the first place. The format string here has no positions for arguments, but one argument was provided.